src/validations/ie/states/go.ts

```diff
@@ -5,9 +5,9 @@ export function validateGO(ie: string): boolean {
 
   if (length !== 9) return false;
 
-  const beginWith = ['10', '11', '15'];
+  const beginRegex = /^(1[015]|2[0-9])/;
```

Comment:
I looked on the link that you put on the PR description. I think it needs an adjust here. 

> Formato da Inscrição: AB.CDE.FGH-I
8 dígitos (ABCDEFGH) + 1 dígito verificador (I); onde AB pode ser igual a 10 ou 11 ou 20 a 29.

But acordingly to your change the code 15 for AB would be accepted
```suggestion
  const beginRegex = /^(1[01]|2[0-9])/;
```